spec/lib/jobs/ansible-job-spec.js

```diff
@@ -3,8 +3,7 @@
 
 'use strict';
 
-var uuid = require('node-uuid'),
-    events = require('events');
+var uuid = require('node-uuid');
 
```

Comment:
Seems this file change doesn't belong to this commit as not relation with kvm. How about move it to another commit?